Sources/MongoSwift/ConnectionString.swift

```diff
@@ -2,6 +2,35 @@ import CLibMongoC
 
 /// Class representing a connection string for connecting to MongoDB.
 internal class ConnectionString {
+    /// Type defining the relationship between a field on MongoClientOptions and a URI option.
```

Comment:
We were doing this in a few places so I defined some abstractions to make it easier to do this without a lot of boilerplate.

Reply:
very nice, cool to put keypaths to use!

![512x512bb](https://user-images.githubusercontent.com/5976377/110017960-caba3f00-7cf4-11eb-8128-1cb547b3040c.jpg)

Reply:
Hah, I appreciate the effort :man_cook: